openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -37,7 +37,12 @@
     attribute_definitions,
 )
 from openpype.lib.attribute_definitions import get_attributes_keys
-from openpype.pipeline import legacy_io, Anatomy
+from openpype.pipeline import (
```

Comment:
This should be tested in Nuke 12 or Maya 2020 as this will probably cause recursion import error.